.changeset/twenty-humans-carry.md

```diff
@@ -0,0 +1,5 @@
+---
+"wrangler": patch
+---
+
+Respect NODE_EXTRA_CA_CERTS env variable for WARP support
```

Comment:
Can we add something here, for internal employees, mentioning the need to run the install-cloudflare-warp-certs script so that we can point people to the changelog

Reply:
is there a wiki we can link to instead? that way if the info changes it will be up to date there.